### PR TITLE
Add SkipUserCreation config to control User and Identity creation

### PR DESCRIPTION
**Pull request by rajivnathan**
Generated files for https://github.com/codeready-toolchain/api/pull/314

e2e tests: https://github.com/codeready-toolchain/toolchain-e2e/pull/571

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -24,4 +24,8 @@ require (
 	sigs.k8s.io/controller-runtime v0.10.3
 )
 
+replace github.com/codeready-toolchain/api => github.com/rajivnathan/api v0.0.0-20220607193200-8c5f18b5d2db
+
+replace github.com/codeready-toolchain/toolchain-common => github.com/rajivnathan/toolchain-common v0.0.0-20220607210046-2133fabb9f02
```
Comment (MatousJobanek) [inline go.mod:29]: reminder

**Review by MatousJobanek — APPROVED** [review]
Nice, but please don't forget to update the e2e test

**Comment by rajivnathan** [comment]
@MatousJobanek thanks for the review. I didn't forget the e2e tests, they just failed locally so I wanted to fix them before pushing them up. I should have mentioned that e2e tests were coming though. Anyway, I think it should be fixed now so I put up https://github.com/codeready-toolchain/toolchain-e2e/pull/571

**Pull request merged**